examples/platforms/simulation/radio.c

```diff
@@ -221,6 +222,54 @@ static uint16_t crc16_citt(uint16_t aFcs, uint8_t aByte)
     return (aFcs >> 8) ^ sFcsTable[(aFcs ^ aByte) & 0xff];
 }
 
+
+#if OPENTHREAD_SIMULATION_VIRTUAL_TIME == 1
+const char * radioStateToString(otRadioState aState)
+{
+    char const * ans;
+    switch (aState)
```

Comment:
```suggestion
    char const * ans;

    switch (aState)
```
nit: add space line between variables and statements to be inline with OT style